scripts/irods/test/test_quotas.py

```diff
@@ -72,7 +72,7 @@ def acRescQuotaPolicy(rule_args, callback, rei):
                         lib.make_file(filename_2, 1024, contents='arbitrary')
                         cmd = 'iput -R {0} {1}'.format(self.testresc, filename_2) # should fail
                         self.admin.assert_icommand(cmd.split(), 'STDERR_SINGLELINE', 'SYS_RESC_QUOTA_EXCEEDED')
-                        cmd = 'istream write nopes'
+                        cmd = 'istream -R {0} write nopes'.format(self.testresc)
```

Comment:
should we keep the non-specific test as well?

Reply:
happy for it to be the way it is - just wanted to bring up coverage.   if you're good, i'm good.